README.md

```diff
@@ -57,3 +57,34 @@ configuration.setVersion("v1");
 // deploy
 deployment.deploy(deployConfiguration);
 ```
+
+## SDK Manager
+
+When using the Cloud SDK manager to provide a cloud SDK to users, the implementer must manage all operations. The operations are intended to run asynchronously, either on an executor or through mechanisms provided by an IDE.
```

Comment:
I don't know what you mean by "must manage all operations."

Reply:
Yeah, I guess what I mean is that it's not updated automatically... I'll change this text.